doc/source/index.rst

```diff
@@ -1,11 +1,11 @@
 .. kiwi documentation master file
 
-KIWI Documentation
-==================
+{kiwi} Documentation
+=====================
 
 .. note::
 
-   This documentation covers KIWI |version|- the command line utility to
+   This documentation covers {kiwi} |version|- the command line utility to
    build Linux system appliances. For versions of KIWI older or equal to
    v7.x.x please refer to `Legacy KIWI <https://doc.opensuse.org/projects/kiwi/doc>`__
```

Comment:
I would rephrase the note to emphasize the difference between the two:


```rst
.. note:: KIWI-NG vs. Legacy KIWI

     This documentation covers {kiwi-full} ({kiwi}) |version| the command line
     utility to build Linux system appliances.
     {kiwi} is stable and all new ideas, bugfixes, and improvements
     will be developed here.
     Older {kiwi} versions older or equal to v7.x.x are out of
     maintenance and do not get any updates or bugfixes.
     If you still need this verson, refer to the documentation for
     Legacy KIWI.
```

Reply:
I like it and used parts of it. Thanks